src/figuresClasses.ts

```diff
@@ -1,19 +1,74 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle',
+  color: 'red' | 'green' | 'blue',
```

Comment:
would be nice to create separate type for this and use it in all places instead of 'red' | 'green' | 'blue'

Reply:
still not resolved 
![image](https://github.com/mate-academy/ts_figures_classes/assets/55200530/90bdc639-5bec-4088-9e6a-df5d7bb5f1b1)

move type color before `Figure` and use it for `color` field